src/main/drivers/camera_control.c

```diff
@@ -69,6 +69,11 @@ void pgResetFn_cameraControlConfig(cameraControlConfig_t *cameraControlConfig)
     cameraControlConfig->internalResistance = 470;
     cameraControlConfig->ioTag = timerioTagGetByUsage(TIM_USE_CAMERA_CONTROL, 0);
     cameraControlConfig->inverted = 0;   // Output is inverted externally
+    cameraControlConfig->buttonResistanceValues[CAMERA_CONTROL_KEY_ENTER] = 45000;
```

Comment:
Not sure we actually need single ohm precision here? We could probably set these in 100 ohm steps, and use `uint16_t` to store them?
(Still, adding `uint32_t` support for arrays is a good thing.)

Reply:
I considered that, but since the current code already has 6810 ohms hardcoded for the "right key" it didn't seem practical.
```
static const int buttonResistanceValues[] = { 45000, 27000, 15000, 6810, 0 };
```
And also some of the use cases were finding they needed about 50ohms for "down" key (currently 0 by default). So the only reasonable other option was to use 10ohm scaling but then that limits the values to about 655K for `uint16` which seemed like a questionable limit.

Reply:
> And also some of the use cases were finding they needed about 50ohms for "down" key (currently 0 by default).

Not sure if 100 ohm would be a close enough approximation for 50 ohm, but considering that the next higher step is 6800 (in 100 ohm steps) one would think so. Do you know who / what hardware this was, so this can be verified?

Reply:
There was some conversation on FB but not sure if people can test. "Measured" values from different camera button pads are all over the place. Caddx cameras seem to want around 1K for the "down" button so this would be ok. But other cases seem to report needing just a little > 0v for the down button. Gets difficult to correlate because in some cases they're measuring voltage directly seen by the camera vs. the static measurements of the button pad (which forms a voltage divider with the internal camera resistance). I guess we could try switching back to `uint16` 100ohm steps and see how it goes. I guess it just depends if it's worth saving 10 bytes vs. just giving the user full control and not have to visit it again.

Reply:
I see your point about not having to visit again. My counterpoint is that not having 1 ohm precision will avoid users trying to make this work with insignificant 1 ohm changes.